src/js/outreach/index.js

```diff
@@ -125,7 +121,7 @@ const FormApp = App.extend({
     dialogView.showChildView('content', new ErrorView());
     this.showView(dialogView);
   },
-  onStart({ actionId }, [form], [definition], [fields]) {
+  onStart({ actionId }, form, definition, fields) {
```

Comment:
@nmajor25 major style change is that `entities` radio requests no longer get returned successfully with the xhr so no need to wrap the arg in an array